ravenframework/Optimizers/GradientDescent.py

```diff
@@ -212,6 +212,7 @@ def __init__(self):
     self._followerProximity = 1e-2   # distance at which annihilation can start occurring, in ?normalized? space
     self._trajectoryFollowers = defaultdict(list) # map of trajectories to the trajectories following them
     self._functionalConstraintExplorationLimit = 500 # number of input-space explorations allowable for functional constraints
+    self._canHandleMultiObjective = False # Currently Gradient Descent cannot handle multiobjective optimization
```

Comment:
I think you do not need to specify it here since it is presented in the base class.